### PR TITLE
Translated privacy.md to French

### PR DESCRIPTION
**Pull request by Utinam-ToO**

### CODE REVIEW

File: fr/privacy.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Le site inclus des options pour sélectionner un thème, un language et une plate-forme de jeu pour chacune de ses pages. Afin de faciliter l'expérience utilisateur, vos préférences sont stockées localement dans un cookie.
+
+Vous trouverez comment effacer ces informations [sur cette page](https://www.privacypolicies.com/blog/how-to-delete-cookies/).
+
+## Analytique
+
+Le site utilise Google Analitics afin d'aider à améliorer son fonctionnement. Google Analytics enregistre votre session sur le site, ce qui inclus les pages que vous visitez, ainsi que le thème, la langue et la plateforme que vous sélectionnez. Ces données sont anonymes et ne peuvent êtres associées avec vous.
+
+Vous pouvez vous désister de l'enregistrement par Google Analytics [avec cette extension de navigateur](https://tools.google.com/dlpage/gaoptout).
```
Comment (EliteMasterEric) [inline fr/privacy.md:20]:
```suggestion
Vous pouvez vous désister de l'enregistrement par Google Analitics [avec cette extension de navigateur](https://tools.google.com/dlpage/gaoptout).
```

Missed one.

File: fr/privacy.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Politique de Confidentialité
+
+Cette page documente comment Krystara.io acquière, stocke et utilise les données collectées auprès des utilisateurs.
+
+## Sommaire
+
+1. [Cookies](#browser-cookies)
+2. [Analytique](#analytics)
+
+## Cookies
```
Comment (EliteMasterEric) [inline fr/privacy.md:10]: Is this correct? Is the term cookies (referring to [HTTP cookies](https://en.wikipedia.org/wiki/HTTP_cookie)) the same between languages?

File: fr/privacy.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Politique de Confidentialité
+
+Cette page documente comment Krystara.io acquière, stocke et utilise les données collectées auprès des utilisateurs.
+
+## Sommaire
+
+1. [Cookies](#browser-cookies)
+2. [Analytique](#analytics)
```
Comment (EliteMasterEric) [inline fr/privacy.md]:
```suggestion
2. [Analytique](#analytique)
```

This needs to change with the header.

**Review by EliteMasterEric — COMMENTED** [review]
Saw a couple issues, created a review. (GitHub is by far the best system for doing this, the reviewing and moderation tools are powerful.

File: fr/privacy.md (hunk 2 above)
Comment (Utinam-ToO) [reply fr/privacy.md:10]: Yes. And "browser" is implied.

File: fr/privacy.md (hunk 1 above)
Comment (Utinam-ToO) [reply fr/privacy.md:20]: Corrected.

File: fr/privacy.md (hunk 3 above)
Comment (Utinam-ToO) [reply fr/privacy.md]: Done.

**Pull request merged**